.github/workflows/build.yml

```diff
@@ -7,8 +7,32 @@ on:
       - litmus-v2
 
```

Comment:
You can also add a path filter at the workflow level itself, So it won't even get triggered when a given directory doesn't have changes. Refer - https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#onpushpull_requestpaths
Something like below - 
```
on:
  pull_request:
    branches:
      - master
      - v*
      - litmus-v2
    paths:
      - 'litmus-portal/**'
```

Reply:
`paths` can be a prerequisite check before the `dorny/paths-filter` action

Reply:
Makes sense. Thanks!